app/src/main/java/io/apicurio/registry/mt/TenantContext.java

```diff
@@ -23,16 +23,22 @@
  */
 public interface TenantContext {
 
-    public static final String DEFAULT_TENANT_ID = "_";
+    String DEFAULT_TENANT_ID = "_";
```

Comment:
why change this from static field to object field? I think this change is not a good idea

Reply:
Because for interfaces, the `public static final` is not needed, since the field is already public, static & final.

Reply:
It can be specified explicitly though, so if you prefer I can leave it there.

Reply:
:facepalm: ohhh I did not know that, sorry, I'm happy to keep it, and thank you!

Reply:
TIL